examples/calendar/backend/privacy_cir/proof.sh

```diff
@@ -9,4 +9,7 @@ snarkjs zkey export verificationkey calendar_0001.zkey verification_key.json
 snarkjs groth16 prove calendar_0001.zkey witness.wtns proof.json public.json
 
 # Verify
-snarkjs groth16 verify verification_key.json public.json proof.json
\ No newline at end of file
+snarkjs groth16 verify verification_key.json public.json proof.json
+
+#create smart contract
+snarkjs zkey export solidityverifier calendar_0001.zkey verifier.sol
```

Comment:
will this `calendar_0001` ever change e.g. for a new calendar? If yes should this not be configurable then in a config file?

Reply:
It's the default file name that's assigned once power of tau is done, not done by me but this file gets generated by circom, so filename_0001.zkey is how they create for 2nd power of tau ceremony, for third it is filename_0002.zkey.